activiti-cloud-services-runtime-bundle/activiti-cloud-services-rest-impl/src/test/java/org/activiti/cloud/services/rest/controllers/TaskControllerImplIT.java

```diff
@@ -282,7 +282,6 @@ public void createNewStandaloneTask() throws Exception {
                                         subsectionWithPath("priority").description("Task priority. Can have values between 0 and 100."),
                                         subsectionWithPath("status").description("Task status (can be " + Arrays.asList(Task.TaskStatus.values()) + ")"),
                                         subsectionWithPath("appName").description("The application name"),
-                                        subsectionWithPath("appVersion").description("The application version"),
```

Comment:
Instead of removing this assert, I think it's better to update the related task so this field is set.